manage.py

```diff
@@ -148,7 +160,8 @@ def get_index_data_from_database(resource="all"):
         test_indexing_search_query(deep_check=False, check_studies=True)
 
     # backup the index data
-    backup_elasticsearch_data()
+    if not nobackup:

```

Comment:
double negative is a bit strange
should the variable be called ``backup`` instead

Reply:
is possible to fix this double negative?